### DISCUSSION TITLE
Not able to send mails to @outlook.com users and  getting  hard failed.

### DISCUSSION THREAD

**Discussion by jaydeepnp** [postalserver/postal#2565] [Help with using Postal]
Hi All,

Need urgent help to resolve my issue for sending mails to @outlook.com , When  mail sent to any user @outlook.com it is getting hardfailed and giving below error.

Permanent SMTP delivery error when sending to outlook-com.olc.protection.outlook.com (104.47.57.33) Recipient added to suppression list (too many hard fails).

550 5.7.1 Unfortunately, messages from [x.x.x.x] weren't sent. Please contact your Internet service provider since part of their network is on our block list (S3140). You can also refer your provider to http://mail.live.com/mail/troubleshooting.aspx#errors. [MW2NAM10FT060.eop-nam10.prod.protection.outlook.com 2023-07-31T08:36:30.580Z 08DB9148005EB1D1]

Time: 0.01s

Support Ref: D34TUCQU

In need of urgent help to resolve this issue.

Regards,
Jaydeep Patel

**Comment by sinnbeck** [comment]
Sounds like your servers ip is blocked by microsoft. You need to ask them to unblock you.

You can try mailing delist@microsoft.com

**Reply by jaydeepnp** [reply]
Hi Sinnbeck,

Thanks for your prompt reply , Could you please help with more details.

mailing delist@microsoft.com with specific information of my mail server or mail to this address from mail server which is blocked ? or I can  send mail from anywhere to this address requesting to unblock my mail server IP ? 

Regards
Jaydeep

**Reply by sinnbeck** [reply]
I have never been blocked myself, but I would use my company email and include the error and the ip in the email. I am sure they will ask you for more information if needed.

**Reply by sinnbeck** [reply]
I can also recommend signing up here, and adding your sender ip's. That way you can track your status in realtime

https://sendersupport.olc.protection.outlook.com/snds

**Reply by jaydeepnp** [reply]
Hi Sinnbeck,

Thanks for reply.

I am checking  on it.

Regards,
Jaydeep
